static/app/components/platformPicker.tsx

```diff
@@ -78,10 +80,11 @@ class PlatformPicker extends React.Component<Props, State> {
 
   logSearch = debounce(() => {
     if (this.state.filter) {
-      analytics('platformpicker.search', {
-        query: this.state.filter.toLowerCase(),
-        num_results: this.platformList.length,
-      });
+      trackAdvancedAnalyticsEvent(
+        'growth.onboarding_platform_search',
```

Comment:
@roggenkemper we don't know this event is part of onboarding so I think `growth.platformpicker_search' is better. Also, we should pass in the `source` here.

Reply:
sounds good! also added the source to the platform categories event and changed that name too